tests/Application/config/sylius/resources.yaml

```diff
@@ -50,3 +50,13 @@ sylius_resource:
             classes:
                 model: App\Entity\PullRequest
                 form: App\Form\Type\PullRequestType
+
+        app.zone:
+            classes:
+                model: App\Entity\Zone\Zone
+                interface: App\Entity\Zone\ZoneInterface
```

Comment:
That'd be cool to configure that interface on existing resource auto registration.